src/metabase/api/dashboard.clj

```diff
@@ -846,6 +847,54 @@
 
 ;;;;;;;;;;;;;;;;;;;;; End functions to handle broken subscriptions
 
+(defn create-editable-table-card!
+  "The implementation details of an editable table dashcard.
+  A quick solution, and a very extensible one, but perhaps overloading a core primitive too much, and could be leaky."
+  [dashboard-id dashcard table-id]
+  (let [tab-id  (:dashboard_tab_id dashcard)
+        table   (t2/select-one [:model/Table :db_id :name] table-id)
+        ;; To keep things simple for now, we don't make any attempt to deduplicate or reuse these cards.
+        ;; This lets us dashboard cards, which helps keep them hidden and have their lifecycle handled implicitly.
+        ;; We need to watch out for other ways they could "leak", for example, in search.
+        ;; We've handled search for now by filtering on :display, but that's quite opinionated and coupled.
+        ;; ;;
+        ;; It does mean that we could create a lot of garbage, but it shouldn't be a big problem in our PoC.
+        ;; We could also change from archiving to deleting them when they are orphaned.
+        card    {:dashboard_id           dashboard-id
+                 :dashboard_tab_id       tab-id
+                 :collection_position    nil
+                 :dataset_query          {:database (:db_id table)
+                                          :query    {:source-table table-id}
+                                          :type     :query}
+                 :description            nil
+                 :display                "table-editable"
```

Comment:
I was surprised that this still results in the FE rendering a table, and it doesn't even log a warning!

This was both confusing and a bit scary.